function-reference/show-alert.md

```diff
@@ -0,0 +1,21 @@
+---
+description: >-
+  Show a temporary notification style alert message to the user
+---
+
+# showAlert
+
+## Signature
+
+```text
+showAlert(message, style) -> void
+```
+
+#### Arguments
+
+| **Argument Name** | **Description** |
+| :--- | :--- |
+| **Message** | string |
+| **Style** | string |
```

Comment:
Please add a more descriptive "Description". What is "Style"? What are the possible values?

Same with "Message" as well. What does it mean? This will be helpful to users reading our docs.

Reply:
okay on it

Reply:
Hey FYI, the style could be one of these strings
```
'info' | 'success' | 'warning' | 'error' | 'default'
```